Makefile

```diff
@@ -219,21 +236,13 @@ clean-slice: ## Removes and cleanups up the systemd slice for the mesos executor
 	@sudo systemctl stop mesos_executors.slice
 	@sudo rm -f $(MESOS_SLICE)
 
-VAGRANT_BOX_VERSION := 8.4.0
-VAGRANT_OVF := $(HOME)/.vagrant.d/boxes/debian-VAGRANTSLASH-jessie64/$(VAGRANT_BOX_VERSION)/virtualbox/box.ovf
-$(VAGRANT_OVF):
-	vagrant box add --provider virtualbox --box-version $(VAGRANT_BOX_VERSION) debian/jessie64
-
-dcos-docker.box: $(VAGRANT_OVF)
-	packer build -var ovf_path=$(VAGRANT_OVF) -var box_output=$@ vagrant/packer.json
-
 clean: clean-certs clean-containers clean-slice ## Stops all containers and removes all generated files for the cluster.
 	$(RM) $(CURDIR)/genconf/ssh_key
 	$(RM) $(CONFIG_FILE)
 	$(RM) -r $(SSH_DIR)
-	$(RM) -r $(SERVICE_DIR)
+	$(RM) -r $(SBIN_DIR)
 	$(RM) dcos-genconf.*.tar
-	$(RM) *.box
+	$(RM) *.box #TODO: remove
```

Comment:
Its more for the future... Current users might have old boxes they don't need any more. 

-Karl

> On Aug 24, 2016, at 12:13 PM, Albert Strasheim <notifications@github.com> wrote:
> 
> In Makefile:
> 
> >  	$(RM) dcos-genconf.*.tar
> > -	$(RM) *.box
> > +	$(RM) *.box #TODO: remove
> @karlkfi do the TODO?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>